fixtures/loader.go

```diff
@@ -384,8 +383,10 @@ func (f *Loader) buildInsertQuery(ctx *loadContext, t string, rows table) (strin
 	for i, field := range fields {
 		fields[i] = "\"" + field + "\""
 	}
-	query := "INSERT INTO \"%s\" (%s) VALUES %s RETURNING row_to_json(\"%[1]s\")"
-	return fmt.Sprintf(query, t, strings.Join(fields, ", "), strings.Join(dbValues, ", ")), nil
+
+	tableAlias := t + "_table_8cc8f4a3" // guarantees that table and column won't collide
```

Comment:
Why not 0xdeadbeaf? :)

Reply:
It could be 0xABADBABE 😄 Couldn't come up with anything original. Any other suggestions? Maybe something related to gonkey?

Reply:
Sure. How "_table_gonkey" looks to you? No magic numbers please :)
And let it be defined by a constant.

Reply:
Done